app/Config/Session.php

```diff
@@ -0,0 +1,105 @@
+use CodeIgniter\Session\Handlers\FileHandler;
+
+class Session extends BaseConfig
+{
+    /**
+     * --------------------------------------------------------------------------
+     * Session Handler
+     * --------------------------------------------------------------------------
+     *
+     * The session storage handler to use:
+     * - `CodeIgniter\Session\Handlers\DatabaseHandler`
+     * - `CodeIgniter\Session\Handlers\FileHandler`
+     * - `CodeIgniter\Session\Handlers\MemcachedHandler`
+     * - `CodeIgniter\Session\Handlers\RedisHandler`
+     *
+     * @var string
+     */
+    public $handler = FileHandler::class;
+
+    /**
+     * --------------------------------------------------------------------------
+     * Session Name
+     * --------------------------------------------------------------------------
+     *
+     * The name of the session which is used as cookie name.
+     * It should only contain alphanumeric characters.
+     *
+     * @var string
+     */
+    public $name = 'ci_session';
+
+    /**
+     * --------------------------------------------------------------------------
+     * Session Life Time
+     * --------------------------------------------------------------------------
+     *
+     * The number of SECONDS you want the session to last.
+     * Setting to 0 (zero) means expire when the browser is closed.
+     *
+     * @var int
+     */
+    public $lifetime = 7200;
+
+    /**
+     * --------------------------------------------------------------------------
+     * Session Save Path
+     * --------------------------------------------------------------------------
+     *
+     * The location to save sessions to and is driver dependent.
+     *
+     * For the 'files' driver, it's a path to a writable directory.
+     * WARNING: Only absolute paths are supported!
+     *
+     * For the 'database' driver, it's a table name.
+     * Please read up the manual for the format with other session drivers.
+     *
+     * IMPORTANT: You are REQUIRED to set a valid save path!
+     *
+     * @var string
+     */
+    public $savePath = WRITEPATH . 'session';
+
+    /**
+     * --------------------------------------------------------------------------
+     * Session Match IP
+     * --------------------------------------------------------------------------
+     *
+     * Whether to match the user's IP address when reading the session data.
+     *
+     * NOTE: If you're using the database driver, don't forget to update
+     *       your session table's PRIMARY KEY when changing this setting.
+     *
+     * @var bool
+     */
+    public $matchIP = false;
+
+    /**
+     * --------------------------------------------------------------------------
+     * Session Time to Update
+     * --------------------------------------------------------------------------
+     *
+     * How many seconds between CI regenerating the session ID.
+     *
+     * @var int
+     */
+    public $cooldown = 300;
```

Comment:
I'm not sure `cooldown` is proper naming. 
At least I don't understand cooldown means session id regenerating time.

Reply:
If we want to be consistent with Cache this would be "ttl" (time to live).

Reply:
i'll go with `$ttl`